deploy/vagrant/provision/playbook.yml

```diff
@@ -3,7 +3,7 @@
   tasks:
     - include: roles/common/tasks/main.yml
     - include: roles/lib/tasks/mount.yml
-      when: provider == "aws"
+      when: (provider == "aws") or (provider == "google")
```

Comment:
The mount task is originally designed specifically for aws ec2, I have no experience with GCE, have you verified that it works?